doc/api/child_process.markdown

```diff
@@ -164,11 +165,11 @@ a `'message'` event on the child.
 
 For example:
 
-    var cp = require('child_process');
+    const cp = require('child_process');
```

Comment:
should this example be made similar to the above?

Specifically making the variable ```const fork = require('child_process').fork;``` and making the returned instance a ```const```